dashboard/app/models/follower.rb

```diff
@@ -42,17 +42,9 @@ def student_cannot_be_admin
     errors.add(:student_user, 'cannot be admin') if student_user.admin?
   end
 
-  def pl_participant_cannot_have_family_name
-    return unless section && student_user
-    if section.pl_section? && student_user.family_name
-      errors.add(:student_user, 'cannot have family_name as a PL participant')
-    end
-  end
-
```

Comment:
@TurnerRiley this is your change, right?

Reply:
Yes!